addons/website/static/src/scss/website.scss

```diff
@@ -1066,12 +1066,27 @@ header {
 } @else if o-website-value('header-template') == 'sidebar' {
     @include media-breakpoint-up(lg) {
         #wrapwrap {
-            // Hack: padding is used by layout option (boxed, etc) so use
-            // border here to be able to combine the effect.
-            @if $-hamburger-right {
-                border-right: o-website-value('sidebar-width') solid transparent;
-            } @else {
-                border-left: o-website-value('sidebar-width') solid transparent;
+            // Hack: padding is also used by layout option (boxed, etc), need to
+            // combine both options/effects...
+            &:not(.o_safari_browser) {
+                @if $-hamburger-right {
+                    border-right: o-website-value('sidebar-width') solid transparent;
+                } @else {
+                    border-left: o-website-value('sidebar-width') solid transparent;
+                }
+            }
```

Comment:
I actually preferred your old version where you did not change this and just reset the border for safari:
- Doing this, you increase the specificity of this border rule, potentially breaking custo
- Even in master: `:not()` rules in CSS are to be avoided when we can (we always can but sometimes it helps):
    - They are inefficient (ok probably neglieable but, still). CSS selectors are computed from right to left, so here the browser asks for each element: "does this have the safari_browser class?" (whose answer is "no" for most elements) then ask "is this the #wrapwrap?").
    - This makes custo more difficult to make (if you want to customize the behavior here, you have to repeat this `not` selector to keep the same specificity for the selector)
    - This is less readable IMO, especially in that case (I would prefer the safari problem be confined to `&.o_safari_browser {` below rather than spread in the "normal" case too :wink: